plugins/CoreHome/lang/en.json

```
@@ -123,6 +123,7 @@
         "SoftwareSubcategoryHelp": "The Software section shows the operating systems, browsers and plugins that your visitors are using to access the site so that you can optimise your site to ensure it is fully compatible with the most popular configurations.",
         "EngagementSubcategoryHelp1": "The Engagement section provides reports that help to quantify how many new and returning visitors you get. You can also review reports that break down the average time and number of pages per visit, as well as the number of times a visitor has been to your site and the most common number of days between visits.",
         "EngagementSubcategoryHelp2": "This can help you to optimise for frequency and high-interaction visits in addition to maximising your reach.",
-        "PeriodHasOnlyRawData": "It looks like reports for this period have not been processed yet. Do you want to see what's happening now? Check out the %1$sVisits log%2$s or choose a different date period until the reports are generated."
+        "PeriodHasOnlyRawData": "It looks like reports for this period have not been processed yet. Do you want to see what's happening now? Check out the %1$sVisits log%2$s or choose a different date period until the reports are generated.",
+        "InternetExplorer11Warning": "Starting in version 4.6, Matomo will be dropping support for Internet Explorer, please upgrade to a more modern browser. For more information %1$ssee our blogpost.%2$s"
```

```suggestion
        "InternetExplorer11Warning": "Starting in version Matomo 4.6, Matomo will be ending support for Internet Explorer. For more information %1$ssee our blog post.%2$s"
```

In the blog post we decided to use the wording "ending support".